src/components/NavBar/NavBar.component.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Styled from './styled';
+
+const NavBar = () => {
+    return (
+        <Styled.NavBar>
+            <Styled.SearchField placeholder="search..."></Styled.SearchField>
+            
+            <Styled.CheckBoxWrapper >
+                <Styled.CheckBox id="checkbox" type="checkbox" />
+                <Styled.CheckBoxLabel htmlFor="checkbox" />
+            </Styled.CheckBoxWrapper>
+
+            <Styled.LoginButton >Login</Styled.LoginButton>
+        </Styled.NavBar>
+    )
+}
```

Comment:
This looks great, you can go one step further and do something like:

```jsx
import {
  NavBar,
  SearchField,
  CheckBoxWrapper,
 ...
} from './Styled';
```

this makes the code more readable

Reply:
great! thanks for the suggestion